app/scripts/HorizontalMultivecTrack.js

```diff
@@ -1,4 +1,5 @@
 import { format } from 'd3-format';
+import equal from 'fast-deep-equal';
```

Comment:
Grumble grumble, another import.

FWIW, unless speed is of the utmost importance (i.e. the operation is being performed multiple times on every redraw) `JSON.stringify(obj1) === JSON.stringify(obj2)`  is usually fast enough. God help you if you have a circular structure though.

Reply:
I have updated this to remove the import